app/src/main/java/ch/epfl/sdp/game/StartGameController.java

```diff
@@ -26,23 +26,28 @@
 
 import static android.content.ContentValues.TAG;
 
-public interface StartGameController {
+public abstract class StartGameController {
+    private RandomEnemyGenerator randomEnemyGenerator;
+    private static final int MAX_ENEMY = 10;
```

Comment:
perhabs you can write down that its in meter the distance.

Reply:
yeah